### DISCUSSION TITLE
Bug: can`t read from /proc/cmdline

### DISCUSSION THREAD

**Discussion by KrasNIX** [vlang/v#14358] [General]
import os

fn main() {
  cmdline := os.read_file('/proc/cmdline') or {
        println('Failed to open file')
        return
    }
  println('cmd=$cmdline')
}

This returns nothing... How read system files form /proc/...?

**Comment by larpon** [comment]
You sure it returns *nothing*?

You have two print statements - one of them should show?

**Reply by changrui** [reply]
Output: `cmd=`.

**Comment by larpon** [comment]
Confirmed.
I've opened a bug report: https://github.com/vlang/v/issues/14361
